src/__tests__/createAuthUser.test.js

```diff
@@ -21,6 +21,7 @@ describe('createAuthUser: basic tests', () => {
       clientInitialized: false,
       email: null,
       emailVerified: false,
+      tenantId: null,
```

Comment:
Please add two tests for each input (one test with a tenant ID, one test without):
* Using firebaseUserClientSDK as input
* Using firebaseUserAdminSDK as input
* Using serializedAuthUser as input